fiaas_deploy_daemon/deployer/kubernetes/ready_check.py

```diff
@@ -66,6 +68,8 @@ def _deployment_ready(self):
                 dep.status.observedGeneration >= dep.metadata.generation)
 
     def _is_certificate_ready(self, cert):
+        if cert.status.NotAfter < datetime.utcnow():
```

Comment:
Will this fail if `notAfter` is null? I think it can be null according to [docs](https://cert-manager.io/docs/reference/api-docs/)
I think we should also return `false` if `notAfter` is null

Reply:
notAfter is optional. That means it could be null, and that could mean that it doesn't expire, so I'm afraid failing with null could lead to false negatives...

Reply:
Also note the default value will be None, and it doesn't fail when it's not set (not with an error, it just takes it as false)

Reply:
I added a specific test for this case